projects/admin/src/app/menu/service/library-switch-menu-storage.service.ts

```diff
@@ -90,7 +87,5 @@ export class LibrarySwitchMenuStorageService {
     this._librarySwitchService.librarySwitch$.subscribe((user: User) => {
       this._localeStorageService.set(this.STORAGE_KEY, this._dataStorage(user));
     });
-    // Delete locale storage on logout
-    this._menuService.logout$.subscribe(() => this.removeStorage())
```

Comment:
This will keep also other possible value from localStorage (fixed checkout date, ...). Is it the expected behavior ?

Reply:
@zannkukai I tested it and it does not seem to keep the fixed date. Maybe the fixed date data is manually deleted on logout?

Reply:
@zannkukai If you need to delete things in locale storage, you need to connect to the obervable $logout. Here I've removed this line, as I need to keep the last library used.